-1

**Failed Tests**:   ClangBuild 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-54febf/11955/summary.html
**CMSSW**: CMSSW_11_3_X_2021-01-04-1100/slc7_amd64_gcc900

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
 - #32048 @jpata: MLPF: first integration of EDProducer and training code

You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-54febf/11955/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-54febf/11955/git-merge-result



## Clang Build

I found compilation error while trying to compile with clang. Command used:
```
USER_CUDA_FLAGS='--expt-relaxed-constexpr' USER_CXXFLAGS='-Wno-register -fsyntax-only' scram build -k -j 32 COMPILER='llvm compile'

```


<pre>>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-01-04-1100/src/CondFormats/SiPixelTransient/src/SiPixelGenError.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-01-04-1100/src/CondFormats/SiPixelTransient/src/SiPixelTemplate2D.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-01-04-1100/src/CondFormats/SiPixelTransient/src/SiPixelTemplate.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-01-04-1100/src/CondFormats/SiPixelTransient/src/SiPixelUtils.cc
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-01-04-1100/src/CondFormats/SiPixelTransient/src/SiPixelGenError.cc:32:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-01-04-1100/src/CondFormats/SiPixelTransient/interface/SiPixelGenError.h:122:15: error: excess elements in scalar initializer
        fbin_{{0, 0, 0}},
              ^ ~~~~~~~
1 error generated.
gmake: *** [tmp/slc7_amd64_gcc900/src/CondFormats/SiPixelTransient/src/CondFormatsSiPixelTransient/SiPixelGenError.cc.o] Error 1
>> Entering Package RecoParticleFlow/Configuration
</pre>

--- a/CondFormats/SiPixelTransient/interface/SiPixelGenError.h
+++ b/CondFormats/SiPixelTransient/interface/SiPixelGenError.h
@@ -112,10 +112,18 @@ struct SiPixelGenErrorStore {  //!< template storage structure
 // ******************************************************************************************
 class SiPixelGenError {
 public:
-  SiPixelGenError(const std::vector<SiPixelGenErrorStore>& thePixelTemp) : thePixelTemp_(thePixelTemp) {
-    id_current_ = -1;
-    index_id_ = -1;
-  }  //!< Constructor for cases in which template store already exists
+  SiPixelGenError(const std::vector<SiPixelGenErrorStore>& thePixelTemp)
+      : id_current_(-1),
+        index_id_(-1),
+        lorxwidth_(0),
+        lorywidth_(0),
+        lorxbias_(0),
+        lorybias_(0),
+        fbin_{0, 0, 0},
+        xsize_(0),
+        ysize_(0),
+        zsize_(0),
+        thePixelTemp_(thePixelTemp) {}  //!< Constructor for cases in which template store already exists
 
   // Load the private store with info from the file with the index (int) filenum from directory dir:
   //   ${dir}generror_summary_zp${filenum}.out

